pkg/broadcastwriter/broadcastwriter_test.go

```diff
@@ -13,6 +13,12 @@ type dummyWriter struct {
 	failOnWrite bool
 }
 
+func compare(t *testing.T, dw *dummyWriter, value string) {
```

Comment:
I'd call this `equal` or something like this, `compare` could mean any type of comparison..